include/odp/api/spec/packet_io.h

```diff
@@ -203,6 +214,13 @@ typedef struct odp_pktin_queue_param_t {
 	  * value is ignored. */
 	odp_queue_param_t queue_param;
 
+	/** Queue parameters override
+	  *
+	  * Overrides queue_param with per queue values. Must point to
+	  * an array of num_queues elements or set to NULL to disable
+	  * queue parameters override. The default value is NULL.
+	  */
+	odp_pktin_queue_param_ovr_t *queue_param_ovr;
```

Comment:
A bit more specification is needed how things are overridden: "When the override array is defined, the same parameter value in 'queue_param' is ignored and these per queue parameter values are used instead. Array elements are used in order (i.e. the first queue gets parameters from the first array element, etc)."

Reply:
OK